tests/integrationtest/r/expression/json.result

```diff
@@ -727,7 +727,7 @@ Error 3146 (22032): Invalid data type for JSON data in argument 1 to function js
 SELECT JSON_OVERLAPS('{}', 123);
 Error 3146 (22032): Invalid data type for JSON data in argument 2 to function json_overlaps; a JSON string or JSON type is required.
 SELECT 'abc' MEMBER OF(123);
-Error 3146 (22032): Invalid data type for JSON data in argument 2 to function member of; a JSON string or JSON type is required.
+Error 3146 (22032): Invalid data type for JSON data in argument 2 to function json_memberof; a JSON string or JSON type is required.
```

Comment:
Looks like this error need to be fixed.

MySQL v8.4.0:
```
mysql-8.4.0> SELECT 'abc' MEMBER OF (123);
ERROR 3146 (22032): Invalid data type for JSON data in argument 2 to function member of; a JSON string or JSON type is required.
```

This is a weird function syntax wise...

Reply:
Done.